CSS/components/input.css

```diff
@@ -0,0 +1,28 @@
+.input-form{
+    display: flex;
+    width: 20rem;
+    height: max-content;
+    flex-direction: column;
+}
+.input-lable{
+    display: block;
+}
```

Comment:
You might want to check this classname.

Reply:
oh ya thank you so much Masoom It's a  typo!!!!

Reply:
corrected it